README.md

```diff
@@ -119,6 +119,11 @@ This repository contains a set of PowerShell cmdlets for developers and administ
 * Azure Automation
   * Manage automation accounts
   * Manage automation jobs, runbooks and schedules
+* Logic Apps
```

Comment:
[Fixed] : Section removed from the Readme.md

From: Mark Cowlishaw [mailto:notifications@github.com]
Sent: Monday, February 1, 2016 1:56 PM
To: Azure/azure-powershell <azure-powershell@noreply.github.com>
Cc: Pankaj Singh Negi <Pankaj.Negi@microsoft.com>
Subject: Re: [azure-powershell] Azure Logic App PowerShell Cmdlets (#1753)


In README.md<https://github.com/Azure/azure-powershell/pull/1753#discussion_r51487428>:

> @@ -119,6 +119,11 @@ This repository contains a set of PowerShell cmdlets for developers and administ

>  * Azure Automation

>    * Manage automation accounts

>    * Manage automation jobs, runbooks and schedules

> +* Logic Apps

Please remove, we are no longer updating the README for new RPs

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-powershell/pull/1753/files#r51487428>.